internal/docsgenerator/lease-metrics.json

```diff
@@ -0,0 +1,19 @@
+[
+    {
+        "MetricName": "kube_lease_owner",
```

Comment:
I assume these are hand written? If yes, then ideal would be if we could get these directly from each resource' FamilyGenerator struct -> https://github.com/kubernetes/kube-state-metrics/blob/master/internal/store/lease.go#L37-L40

Reply:
Working on this